libs/extensions/angular/image-banner/src/image-banner.component.stories.ts

```diff
@@ -107,6 +122,49 @@ export const NoDismiss: Story = {
   }),
 };
 
+const setCustomContent = (args: any) => {
+  return {
+    ...args,
+    customTitle: args.title,
+    title: null,
+    customBodyText: args.bodyText,
+    bodyText: null,
+  };
+};
+
+/**
+ * The title and body text can be truncated with an ellipsis when they exceed the width of the banner.
+ * The banner can be set to have a minimum height to prevent it from shrinking too much.
+ */
+export const customContent: Story = {
+  args: {
+    title:
+      'The title is very long and will NOT be truncated with an ellipsis when it exceeds the width of the banner.',
+    bodyText:
+      'A lot of text that should be truncated with an ellipsis when it exceeds the width of the banner.',
```

Comment:
I think it would be better for the readability of the example if you add this text directly inline in the template instead, and just remove these two properties (`title` and `bodyText`). Then we also validate that the content projection works!